sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/FetchedGeofence.kt

```diff
@@ -0,0 +1,41 @@
+package com.klaviyo.analytics.networking.requests
+
+import com.klaviyo.core.Registry
+import org.json.JSONArray
+import org.json.JSONObject
+
+/**
+ * Represents a geofence from the API response with its attributes.
+ */
+data class FetchedGeofence(
```

Comment:
if the backend is already adding dwell would it make sense to also include it here from the start to avoid any potential migration we may have to do in the future if this gets persisted to disk etc?

Reply:
I'd like to address Dwell events in a standalone PR, as for timing, yeah ideally we'd handle that before the first LA.